cmake/zstd.cmake

```diff
@@ -23,30 +23,6 @@
 # cmake -DWITH_ZSTD=system|bundled
 # bundled is the default
 
-<<<<<<< HEAD
-# zstd 1.0.0 is required for: ZSTD_CStream, ZSTD_DStream, ZSTD_createCStream, ZSTD_createDStream, ZSTD_inBuffer, ZSTD_outBuffer
-SET(LIBZSTD_VERSION_REQUIRED "1.0.0")
-
-MACRO (CHECK_ZSTD_VERSION)
```

Comment:
Likewise, why `CHECK_ZSTD_VERSION()` removed?

Reply:
The same as above. fixed.